components/renku_data_services/storage_models/core.py

```diff
@@ -15,17 +17,117 @@ class CloudStorage:
 
     storage_id: str | None = None
 
+    source_path: str
+    target_path: str
```

Comment:
I think it would be nice make the type for the source and target path `Path` from the `pathlib` library. Or at least should we add some validation to make sure they are proper valid paths?

Reply:
`Path` doesn't really validate the code and validation fails if the folder doesn't exist (which it does not).

I've added validation in a "unify databases" follow-up PR